### ISSUE TITLE
trading for "Declare war" Issue

### ISSUE THREAD

**Issue by lorduc** [LoneGazebo/Community-Patch-DLL#3360]
_1. Mod version (i.e Date - 4/23):_

July 15th (7/15) (the one with "prize rules" available to all naval units issue.
_2. Mod list (if using Vox Populi only, leave blank):_
VP no EUI

_3. Error description:_ 
 
Might be a mod conflict but I thought I should bring it to your attention anyway.
Its a game with 8 civs. If I click on any of Arabia's city, trade > declare war on > Washington> impossible>click on cancel instead of propose. 
Then if I go to America's trade screen> declare war> there's no arabia option. 
basically if I talk to any civ, then I cant ask other civs to declare war on them, the option is simply not there. city states also disappear for no reason. If I talk to all civs, (simply going into their leader screen) then that means I can't ask any civ to declare war or make peace with another civ (from the trade screen, not discuss screen, I havent tried discuss screen)
saving the game and reloading the save restores the options for all civs and city states, however doesnt stop it from happening again.

**Comment by LoneGazebo** [comment]
Well the lists change based on people the AI you ask stuff has met.

**Comment by lorduc** [comment]
But Arabia is next to Washington they have mutual borders and even shared religion. If I ask Arabia for a war declare on washington and then go to Washington leader screen then back to arabia, Washington will disappear as an option...

**Comment by LoneGazebo** [comment]
I'll look.

**Comment by LoneGazebo** [comment]
Rabbit hole discovered, fixed bug. Thanks for the diligent reporting.
